test/functional/message/message-formatter.js

```diff
@@ -175,6 +178,13 @@ QUnit.test( "should support ICU message format", function( assert ) {
 	assert.equal( like({ count: 3 }), "You and 2 others liked this" );
 });
 
+QUnit.test( "should support Bidi structured text", function( assert ) {
+	assert.equal(
+		Globalize( "he" ).messageFormatter( "helloArray", true )( "Beethoven", "Mozart" ),
+		"\u05e9\u05dc\u05d5\u05dd, \u200FBeethoven\u200F & \u200FMozart\u200F"
```

Comment:
Can we replace `\u05e9\u05dc\u05d5\u05dd` with real characters like `שלום` and do the same for all other visual characters?